integrations/huggingface-transformers/README.md

```diff
@@ -34,87 +30,156 @@ Coming soon!
-  --output_dir MODELS_DIR/bert-base-12layers_prune80 \
-  --cache_dir cache \
-  --preprocessing_num_workers 6 \
-  --fp16 \
-  --num_train_epochs 30 \
-  --recipe recipes/bert-base-12layers_prune80.md \
-  --onnx_export_path MODELS_DIR/bert-base-12layers_prune80/onnx \
-  --save_strategy epoch \
-  --save_total_limit 2
+sparseml.transformers.question_answering \
+    --output_dir models/teacher \
+    --model_name_or_path zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none \
+    --recipe zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none?recipe_type=transfer-question_answering \
+    --dataset_name squad \
+    --per_device_train_batch_size 16 \
+    --per_device_eval_batch_size 24 \
+    --preprocessing_num_workers 6 \
+    --do_train \
+    --do_eval \
+    --evaluation_strategy epoch \
+    --fp16 \
+    --seed 42 \
+    --per_device_train_batch_size 16 \
+    --per_device_eval_batch_size 24 \
+    --save_strategy epoch \
+    --save_total_limit 1
 ```
 
-### Structure
+With the dense teacher trained to convergence, you can begin the sparse transfer learning with distillation with a recipe. The dense teacher will distill knowledge into the sparse architecture, therefore increasing its performance while ideally converging to the dense solution’s accuracy.
 
-The following table lays out the root-level files and folders along with a description for each.
+💡**PRO TIP**💡: Recipes encode the instructions and hyperparameters for sparsifying a model using modifiers to the training process. The modifiers can range from pruning and quantization to learning rate and weight decay. When appropriately combined, it becomes possible to create highly sparse and accurate models.
 
-| Folder/File Name     | Description                                                                                                           |
-|----------------------|-----------------------------------------------------------------------------------------------------------------------|
-| recipes              | Typical recipes for sparsifying NLP models along with any downloaded recipes from the SparseZoo.                      |
-| tutorials            | Tutorial walkthroughs for how to sparsify NLP models using recipes.                                                   |
-| transformers         | Integration repository folder used to train and sparsify NLP models (`setup_integration.sh` must run first).            |
-| README.md            | Readme file.                                                                                                          |
-| setup_integration.sh | Setup file for the integration run from the command line.                                                             |
+Once the command has completed, you will have a sparse checkpoint located in `models/sparse_quantized`.
 
-### Exporting for Inference
+### Quantization-Compatible Deployment Environment
```

Comment:
this is not about a deployment environment, it's about getting a quantized model from a sparse base model?